shared/wallets/link-existing/index.js

```diff
@@ -171,7 +171,7 @@ const EnterKey = (props: EnterKeyProps) => (
 type EnterNameProps = {
   error: string,
   name: string,
-  onBack: ?() => void,
+  onBack?: () => void,
```

Comment:
The same thing goes for entering a name in the link existing popup – sometimes there is a back button and sometimes there isn't. Seems like it'd be better to make it optional instead of maybe to be consistent.